3.processing_CP_features/3.preprocessing_CFReT_features.yml

```diff
@@ -0,0 +1,17 @@
+name: 3.process-cfret-features
```

Comment:
I noticed several variations of the names used for the environment, the folder, and these files. Would it make sense to unify or adjust the names to increase readability?

For reference:
- Environment name: `3.process-cfret-features`
- File name: `3.preprocessing_CFReT_features.yml`
- Folder name: `3.processing_CP_features`
- Notebook name: `extract_sc_features.ipynb` (uses `3.process-cfret-features` environment)

Reply:
this is a good catch @d33bs! Yes, I think it does make sense to unify naming!

Reply:
You are right, I didn't even notice! I will leave the notebook different since the it should be more descriptive imo. I have changed the rest. Thank you!